### PR TITLE
feat(core): add SourceGovernor to hardhat deployment

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

To do the L1 deployments for the l1-l2 bridge, we need a source governor script.

**Summary**

Added this deployment script, and updated a few minor tags/dependencies for clarity.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

N/A

### CODE REVIEW

File: packages/core/deploy/015_deploy_sink_governor.js [hunk 1]
```diff
@@ -13,5 +13,5 @@ const func = async function (hre) {
   });
 };
 module.exports = func;
-func.tags = ["SinkGovernor", "sink-governor"];
-func.dependencies = ["Finder"];
+func.tags = ["SinkGovernor", "bridge-l2"];
```
Comment (nicholaspai) [inline packages/core/deploy/015_deploy_sink_governor.js:16]: These tags make sense. Would you mind adding them to some sort of readme?

hardhat-deploy tags:
- `dvm`: deploys core voting system infrastructure
- `bridge-l2`: deploys infrastructure to L2 to enable cross-EVM communication.
- `bridge-l1`: deploys infrastructure to L1 to enable cross-EVM communication

**Review by nicholaspai — APPROVED** [review]
LGTM

File: packages/core/deploy/015_deploy_sink_governor.js (hunk 1 above)
Comment (mrice32) [reply packages/core/deploy/015_deploy_sink_governor.js:16]: Do we have a readme for hardhat deployment at all?

Comment (nicholaspai) [reply packages/core/deploy/015_deploy_sink_governor.js:16]: Sounds good to me

**Pull request merged**